Packs/RsaNetWitnessPacketsAndLogs/ReleaseNotes/1_0_5.md

```diff
@@ -1,8 +1,12 @@
-
 #### Integrations
 
 ##### RSA NetWitness Packets and Logs
 
 - You can now use credentials when configuring the following fields:
   - *Username*
   - *Password*
+- change logic for SSL, the port doesn't define the activation of SSL
+- add parameters suggestions
+
+###### netwitness-packets
+- Add parameter (renderToContext) to output the result in the contextdata
```

Comment:
Additionally, please update the arguments you added to each command in this standard:
https://xsoar.pan.dev/docs/documentation/release-notes#examples,
for example:
- Added support for the *extend-context* argument in the ***ua-parse*** command.

Reply:
please add to the rn: 

Added support for the *useSSL* parameter in all commands, allowing the selection of SSL with every port.